dev-util/plan9port/plan9port-20200330.ebuild

```diff
@@ -0,0 +1,132 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing toolchain-funcs
+
+MY_HASH="4650064aa757c217fa72f8819a2cf67c689bcdef"
+MY_P="${PN}-${MY_HASH}"
+
+DESCRIPTION="Port of many Plan 9 programs and libraries"
+HOMEPAGE="https://9fans.github.io/plan9port/"
+SRC_URI="https://github.com/9fans/${PN}/archive/${MY_HASH}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="9base BSD-4 MIT LGPL-2.1 BigelowHolmes"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="X aqua truetype"
+REQUIRED_USE="?? ( X aqua )"
+
+DEPEND="
+	X? ( x11-apps/xauth )
+	truetype? (
+		media-libs/freetype
+		media-libs/fontconfig
+	)
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-noexecstack.patch"
+	"${FILESDIR}/${PN}-cflags.patch"
+	"${FILESDIR}/${PN}-builderr.patch"
+)
+
+S="${WORKDIR}/${MY_P}"
+
+PLAN9="/usr/lib/plan9"
```

Comment:
How does this work in a multilib system?

Reply:
I don't know, I don't have a multilib system to test it on. I think it should be sufficient to use get_libdir here, or just use /opt instead.